yarn.lock

```diff
@@ -19241,7 +19241,7 @@ jest@^29.3.1:
     import-local "^3.0.2"
     jest-cli "^29.3.1"
 
-joi@^17.3.0, joi@^17.4.0:
+joi@^17.3.0:
   version "17.4.0"
```

Comment:
The remaining v17.4.0 instance of `joi` should also be upgraded in this PR as it's within the allowed semver range. The way to do that would be to simply delete this section from the `yarn.lock` file and run:

```
yarn cache clean joi && yarn kbn bootstrap
```

That should ensure that yarn knows about the very latest version of joi and will update the lock file to match.

Reply:
Whoops! Thanks! I actually had to manually delete it, but I think it's right now.